src/pages/AddPersonalBankAccountPage.tsx

```diff
@@ -52,7 +53,7 @@ function AddPersonalBankAccountPage({personalBankAccount, plaidData}: AddPersona
             } else if (shouldContinue && onSuccessFallbackRoute) {
                 PaymentMethods.continueSetup(onSuccessFallbackRoute);
             } else {
-                Navigation.goBack();
+                Navigation.navigate(ROUTES.SETTINGS_WALLET);
```

Comment:
I'm not sure I understand this change, can you explain?

Reply:
@daledah Bump on this question, please :pray:

Reply:
+1